### PR TITLE
Improved the main thread activity categorization

### PR DESCRIPTION
**Pull request by pmeenan**
This adds a few new categories that are showing up, adds a blanket categorization for all v8.* events and alphabetizes each group of events.

### CODE REVIEW

File: www/breakdownTimeline.php [hunk 1]
```diff
@@ -31,52 +31,63 @@
...
+                               'MinorGC' => 'Scripting',
+                               'ThreadState::completeSweep' => 'Scripting',
+                               'ThreadState::performIdleLazySweep' => 'Scripting',
                                'TimerFire' => 'Scripting',
-                               'EventDispatch' => 'Scripting',
                                'TimerInstall' => 'Scripting',
                                'TimerRemove' => 'Scripting',
+                               'v8.compile' => 'Scripting',
+                               'XHRLoad' => 'Loading',
                                'XHRLoad' => 'Scripting',
```
Comment (stoyan) [inline www/breakdownTimeline.php:51]: duplicate?

File: www/breakdownTimeline.php [hunk 2]
```diff
@@ -31,52 +31,63 @@
...
+                               'IntersectionObserverController::computeIntersections' => 'Scripting',
+                               'MajorGC' => 'Scripting',
+                               'MinorGC' => 'Scripting',
+                               'ThreadState::completeSweep' => 'Scripting',
+                               'ThreadState::performIdleLazySweep' => 'Scripting',
                                'TimerFire' => 'Scripting',
-                               'EventDispatch' => 'Scripting',
                                'TimerInstall' => 'Scripting',
                                'TimerRemove' => 'Scripting',
+                               'v8.compile' => 'Scripting',
```
Comment (stoyan) [inline www/breakdownTimeline.php:49]: this is taken care of later in `stripos($type, 'v8.')`, right?

Comment (pmeenan) [reply www/breakdownTimeline.php:49]: Yep.  Left it just for legacy reasons but happy to just remove it

**Review by stoyan — APPROVED** [review]
Awesome, thanks! Love the A-Z :)

**Review by tkadlec — APPROVED** [review]

**Pull request merged**